webrender/src/tiling.rs

```diff
@@ -818,6 +818,42 @@ enum AlphaBatchKind {
     BoxShadow,
 }
 
+bitflags! {
+    pub flags AlphaBatchKeyFlags: u8 {
+        const AXIS_ALIGNED    = 0b00000010,
+        const COMPLEX         = 0b00000100,
+        const NEEDS_CLIPPLING = 0b00000001,
+    }
+}
+
+impl AlphaBatchKeyFlags {
+    fn new(transform_kind: TransformedRectKind,
+        needs_clipping: bool) -> AlphaBatchKeyFlags {
```

Comment:
Though now seems easier to just use the literals and or them as you've done below, so we might as well get rid of this function entirely.